src/ConfigFile.c

```diff
@@ -572,15 +575,32 @@ void Ros_ConfigFile_Parse()
         //Parse file
         yaml_parser_t parser;
         yaml_event_t event;
-        char sramFilePath[128];
+        const int CHAR_BUFFER_SIZE = 128;
+        char configFilePath[CHAR_BUFFER_SIZE];
+        char storageDrive[CHAR_BUFFER_SIZE];
         int fd;
         struct stat fileStat;
 
-        sprintf(sramFilePath, "%s\\%s", MP_SRAM_DEV_DOS, CONFIG_FILE_NAME);
+#if (DX200 || FS100)
+        snprintf(storageDrive, CHAR_BUFFER_SIZE, "%s", MP_USB0_DEV_DOS);
+#else //(YRC1000 and newer)
+        snprintf(storageDrive, CHAR_BUFFER_SIZE, "%s", MP_SRAM_DEV_DOS);
+#endif
+
+        snprintf(configFilePath, CHAR_BUFFER_SIZE, "%s\\%s", storageDrive, CONFIG_FILE_NAME);
 
         Ros_Debug_BroadcastMsg("Checking configuration file: %s", CONFIG_FILE_NAME);
 
-        fd = mpOpen(sramFilePath, O_RDONLY, 0);
+        fd = mpOpen(configFilePath, O_RDONLY, 0);
+
+#if (DX200)
+        if (fd < 0)
```

Comment:
Might be nice to add a debug log here.

Reply:
I considered that and then figured that a user wouldn't care and that information would just add noise. Physically, it just looks like a USB hub and a user would just plug in the drive arbitrarily. 
![image](https://github.com/Yaskawa-Global/motoros2/assets/4623811/06900073-643f-409d-9ca8-0887aa5eee99)

Reply:
I haven't checked the changes in detail, but how would the "no usb stick inserted" situation be handled?